plugins/home/backstage-plugin-home-rss/src/RSSCard/types.ts

```diff
@@ -14,11 +14,24 @@
  * limitations under the License.
  */
 
+/**
+ * Element displayed in each row for the RSS Feed {@link Content}.
+ *
+ * @public
+ */
+export type DataItem = {
+  title: JSX.Element;
+};
+
 /**
  * Props for RSS content component {@link Content}.
  *
  * @public
  */
 export type RSSContentProps = {
   feedURL: string;
+  paging?: boolean;
```

Comment:
Would it make sense to add these to the readme file?

Reply:
Indeed it does. I added a few more lines to the README file